website/src/docs/vue.md

```diff
@@ -0,0 +1,151 @@
+---
+title: "Vue Integration"
+type: docs
+module: "@uppy/vue"
+permalink: docs/vue/
+order: 0
+category: "Vue"
+---
+
+Uppy provides [Vue][] components for the included UI plugins.
+
+## Installation
+
+All Vue components are provided through the `@uppy/vue` package
+
+Install from NPM:
+
+```sh
+npm install @uppy/vue
+```
+## CSS
+
+Make sure to also include the necessary CSS files for each Uppy Vue component you are using.
+
+## Usage
+
+The components can be used with [Vue][] and frameworks that use it, like [Nuxt][].
+
+Instead of adding a UI plugin to an Uppy instance with `.use()`, the Uppy instance can be passed into components as an `uppy` prop. 
+
+Note: At the moment, this example is not functional due to an unresolved issue with using other plugins with the Dashboard
+
+```vue
+<template>
+  <div id="app">
+    <Dashboard :uppy="uppy" :plugins="['Webcam']"/>
+  </div>
+</template>
+
+<script>
+import { Dashboard } from '@uppy/vue'
+
+import '@uppy/core/dist/style.css'
+import '@uppy/dashboard/dist/style.css'
+
+import Uppy from '@uppy/core'
+import Webcam from '@uppy/webcam'
+
+export default {
+  name: 'App',
+  components: {
+    Dashboard
+  },
+  computed: {
+    uppy: () => new Uppy().use(Webcam)
```

Comment:
Is it guaranteed that this computed property will only be called once? Or else is there some kind of beforeMount() style method that folks could use to create the instance? It's critical that the same instance is used for the entire lifetime of the component.

Reply:
Because there is no dependency, the Uppy instance *should* only be called once. I believe it's somewhat equivalent to passing an empty array to `useEffect`

Reply:
We could also do something like this
```
// snip
data: {
  uppy: null
},
beforeMount() {
  this.uppy = new Uppy().use(Webcam)
}
```

Reply:
I think that approach is preferable, because it makes clear when exactly the instance is created, and you can immediately see that it maintains the same instance throughout even if you don't know a lot about Vue (like me :laughing: ).